include/LVSS.hpp

```diff
@@ -1,13 +1,59 @@
-#pragma once
+#ifndef _LVSS_
+#define _LVSS_
+
+#include "EVT/dev/LCD.hpp"
+#include "EVT/io/CANopen.hpp"
+#include "EVT/io/GPIO.hpp"
+#include "EVT/io/SPI.hpp"
+#include "EVT/utils/log.hpp"
+#include "LVSS.hpp"
+#include "dev/INA138.hpp"
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace log = EVT::core::log;
 
 namespace LVSS {
 
 /**
  * This is an example of a class for a board
  */
-class LVSS {
+class LVSS : public CANDevice {
 public:
+    static constexpr uint8_t NODE_ID = 42;
+
+    /**
+     * Placeholder constructor for the LVSS class
+     */
+    LVSS(IO::GPIO& lvssEn);
+
+    CO_OBJ_T* getObjectDictionary() override;
+
+    /**
```

Comment:
Because these methods are overridden, they'll inherit Doxygen from their parent class. Remove the comments for this and the following function